### DISCUSSION TITLE
Change app layout

### DISCUSSION THREAD

**Discussion by rackom** [usebruno/bruno#3337] [Ideas]
It would be cool if we can change layout of the application where response-related column is not to the right of the input-related column but rather below it - similar to what Postman has. When using 21:9 monitor split in half for code / Bruno columns are too narrow.

**Comment by rreyn-bruno** [comment]
Hi @rackom - thanks for bringing this up. We'll have a number of UX/UI improvements coming in 2025. Could you take a screenshot of what you're describing and we can create an issue to track this?

**Comment by rackom** [comment]
Here is the Bruno vertical layout and Postman horizontal layout for response pane.
![Screenshot 2024-10-31 at 8 12 48](https://github.com/user-attachments/assets/23a8148d-e37c-4ca5-b003-8d86fda0aac9)
![Screenshot 2024-10-31 at 8 12 03](https://github.com/user-attachments/assets/c2a0c452-bf66-4342-95c5-5c12df3e4778)

**Comment by MrInterBugs** [comment]
Purely curious but any updates on this, its not quite a make or break deal for me but it is very annoying?

**Comment by rreyn-bruno** [comment]
@MrInterBugs @rackom - should have updated, we released this in 2.7.0. You can switch the layout by clicking on the button highlighted in this screenshot. 

<img width="1725" height="974" alt="image" src="https://github.com/user-attachments/assets/d5d0ed6e-8175-44dc-81ea-07f2346a21b2" />

**Reply by MrInterBugs** [reply]
I am clearly blind.. Either way, great to hear, thanks!

**Reply by rreyn-bruno** [reply]
It's definitely a small icon! Glad you're happy with it though.

**Reply by castdrian** [reply]
> It's definitely a small icon! Glad you're happy with it though.

the main issue is that it only shows when you have a response present, if there's none the menu is gone

**Reply by yogesnsamy** [reply]
Using Bruno 3.3.0

Is there a way to switch the  `Variables` to its original position?
BEFORE - `variables` is on the same row as the `Query`, `Headers` etc:
<img width="753" height="543" alt="image" src="https://github.com/user-attachments/assets/1c2ccdb2-0e85-4ef8-a51c-3243ebb809dc" />
NOW - `variables` has moved to its own section and I'm not able to change it:
<img width="2291" height="1083" alt="image" src="https://github.com/user-attachments/assets/70266ec8-0a27-43f0-9dd7-e5d4de4241b7" />

**Reply by anusree-bruno** [reply]
@yogesnsamy We’ve moved it under the query section since users can view both while editing, especially with the query builder in place now. Is there a specific reason you’d prefer it to remain in the tabs?

**Reply by yogesnsamy** [reply]
> @yogesnsamy We’ve moved it under the query section since users can view both while editing, especially with the query builder in place now. Is there a specific reason you’d prefer it to remain in the tabs?

Because I'm using it to test gql functions and now I have to perform extra steps to expand the query section -> reduce the size of the query window -> update variables. It's challenging to view all the info I need when the query or variable is long. I'm attaching a sample video for your reference. https://github.com/user-attachments/assets/44b674b6-439e-43cb-9dc9-b8b665ce5561

I'm not familiar with the query builder. Can you pls share on how it'll help my use-case?

**Reply by anusree-bruno** [reply]
Thanks for sharing the video, @yogesnsamy. We’ll improve visibility to view long queries without resizing.

The Query Builder provides a visual representation of the schema and helps construct queries, but it’s not related to this issue. You can check out the docs here: https://docs.usebruno.com/send-requests/graphql/graphql-api#query-builder
